--- index.theme
+++ index.theme
@@ -4,7 +4,7 @@
 Directories=128x128/actions,128x128/apps,128x128/categories,128x128/devices,128x128/emotes,128x128/mimetypes,128x128/places,128x128/preferences,128x128/status,96x96/actions,96x96/apps,96x96/categories,96x96/devices,96x96/emotes,96x96/mimetypes,96x96/places,96x96/preferences,96x96/status,64x64/actions,64x64/apps,64x64/categories,64x64/devices,64x64/emotes,64x64/mimetypes,64x64/places,64x64/preferences,64x64/status,48x48/actions,48x48/apps,48x48/categories,48x48/devices,48x48/emotes,48x48/mimetypes,48x48/places,48x48/preferences,48x48/status,40x40/actions,40x40/apps,40x40/categories,40x40/devices,40x40/emotes,40x40/mimetypes,40x40/places,40x40/preferences,40x40/status,32x32/actions,32x32/apps,32x32/categories,32x32/devices,32x32/emotes,32x32/mimetypes,32x32/places,32x32/preferences,32x32/status,22x22/actions,22x22/apps,22x22/categories,22x22/devices,22x22/emotes,22x22/mimetypes,22x22/places,22x22/preferences,22x22/status
 
 DisplayDepth=32
-Inherits=hicolor
+Inherits=oxygen,breeze,hicolor
 Example=folder
 FollowsColorScheme=true
 
